packages/react-ui/src/theme/carto-theme.d.ts

```diff
@@ -63,6 +66,7 @@ type CustomPalette = Modify<
     white: CustomShadesPaletteColor;
     black: CustomShadesPaletteColor;
     brand: CustomBrandPaletteColor;
+    custom: CustomPaletteColor;
```

Comment:
I'm not sure if we need here 2 'custom palettes' here, Can't we just use the current brand color in the component @vmilan ?

Reply:
Done!